.github/workflows/prof_controller_manifests_test.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build Profile Controller manifests
+on:
+  pull_request:
+    paths:
+      - components/profile-controller/**
```

Comment:
I think you missed this one. It should be `components/profile-controller/config/**`

Reply:
Yeap you are correct. Fixed it.